sw/device/silicon_creator/rom/data/rom_e2e_testplan.hjson

```diff
@@ -1111,7 +1111,7 @@
             '''
       tags: ["rom", "no_dv", "fpga", "silicon"]
       stage: V2
-      tests: []
+      tests: ["rom_keymgr_init"]
```

Comment:
@alphan, FYI.

Reply:
not sure this will help since this has the tag "no_dv". You will probably have to remove that too?

Also, this the `chip_rom_tests.hjson` entry and Bazel build label for this test is not quite correct, hence is failing in DV. I can fix in a separate PR.

Reply:
This should do the trick: #15974

Reply:
Thanks @sriyerg and @timothytrippel!